portfolio/src/main/webapp/script.js

```diff
@@ -35,3 +35,42 @@ async function showString() {
     const stringContainer = document.getElementById('string-container');
     stringContainer.innerText = textFromResponse;
 }
+
+function createMap() {
+    const map = new google.maps.Map(
+        document.getElementById('map'),
+        {center: {lat: 40.76060063892388, lng:-73.98365302868159}, zoom: 12});
+    
+    const Birch = new google.maps.Marker({
+    position: {lat:40.76406941409203, lng: -73.96651094290358},
+    map: map,
+    title: 'Birch Coffee'
+    });
+
+    const Bluebottle = new google.maps.Marker({
+        position: {lat:40.719883932218984, lng: -74.00210380567277},
+        map: map,
+        title: 'Blue Bottle Coffee'
+        });
+
+        const Variety = new google.maps.Marker({
```

Comment:
another style nit: keep the whitespace consistent. Here and below should be tabbed out.